widget/ui/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,76 @@
+import type { PropTypes } from './Tabs.types';
+
+import React, { useEffect, useRef, useState } from 'react';
+
+import { Divider } from '../Divider';
+import { Tooltip } from '../Tooltip';
+
+import { BackdropTab, Tab, Tabs } from './Tabs.styles';
+
+export function TabsComponent(props: PropTypes) {
+  const {
+    items: tabItems,
+    onChange,
+    container = document.body,
+    value,
+    type,
+    borderRadius,
+  } = props;
+  const [tabWidth, setTabWidth] = useState(0);
+  const tabRef: React.Ref<HTMLButtonElement> = useRef(null);
+  const currentTabIndex = tabItems.findIndex((item) => item.id === value);
+
+  useEffect(() => {
+    const updateTabWidth = () => {
+      if (tabRef.current) {
+        const tabRect = tabRef.current.getBoundingClientRect();
+        setTabWidth(tabRect.width);
+      }
+    };
+    updateTabWidth();
+    window.addEventListener('resize', updateTabWidth);
+    return () => {
+      window.removeEventListener('resize', updateTabWidth);
+    };
+  }, []);
+
+  return (
+    <Tabs type={type} borderRadius={borderRadius}>
+      {tabItems.map((item, index) => (
+        <Tooltip
+          key={item.id}
+          style={{ width: '100%' }}
+          container={container}
+          side="bottom"
+          sideOffset={2}
+          content={item.tooltip}
+          open={!item.tooltip ? false : undefined}>
```

Comment:
```suggestion
          open={item.tooltip || undefined}>
```

Reply:
In this way, Tooltip always remains open and doesn't work properly.
Actually, we want to keep it closed only when we don't have a tooltip.